astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -41,13 +87,30 @@ class galactocentric_frame_defaults(ScienceState):
     ``Galactocentric`` or ``Galactocentric()`` with no explicit arguments.
 
     This class controls the parameter settings by specifying a string name,
-    which can be one of:
+    with the following pre-specified options:
 
     - 'pre-v4.0': The current default value, which sets the default frame
       attribute values to their original (pre-astropy-v4.0) values.
     - 'v4.0': The attribute values as updated in Astropy version 4.0.
     - 'latest': An alias of the most recent parameter set (currently: 'v4.0')
 
+    Alternatively, user-defined parameter settings may be registered, with
+    :meth:`~astropy.coordinates.galactocentric_frame_defaults.register`,
+    and used identically as pre-specified parameter sets. At minimum,
+    registrations must have unique names and a dictionary of parameters
+    with keys "galcen_coord", "galcen_distance", "galcen_v_sun", "z_sun",
+    "roll". See examples below.
+
+    This class also tracks the references for all parameter values in the
+    attribute ``references``, as well as any further information in
```

Comment:
Don't know why I see this only now, but shouldn't we remove the "as well as ... `_state`" sentence? I think you rightly note below that the preferred way is to use `get_from_registry` -- which seems better than exposing a private attribute!

Reply:
changing it to "as well as any further information the registry."